Course/Course/Presentation/Subviews/TopHeaderView.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  TopHeaderView.swift
```

Comment:
How about giving it a more meaningful name like course header view or some as per its functionality Thoughts?

Reply:
Renamed to CourseHeaderView